### DISCUSSION TITLE
Unable to Verify Academic Affiliation

### DISCUSSION THREAD

**Discussion by rbellet** [community/community#56486] [GitHub Education]
### Hi everyone,

I was applying, and asked to complete my profile with my full name and a short bio.
I did those things, and applied two more times, but got the same message:
Hey rbellet, 
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 26, 2023 for [@rbellet](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Frbellet&data=05%7C01%7Crbellet%40ecpi.edu%7C168af546a7cf42795b9408db5df07b5a%7C50656f6f463c40549fc26f4071c6e7fc%7C0%7C0%7C638207059688917082%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=JmQ2tpky9c8sq3RGS%2B6BN5fa5DPWUwSrAGBcFMM4Ue4%3D&reserved=0). 
•	You are unlikely to be verified until you have completed your [GitHub user profile](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.github.com%2Fen%2Faccount-and-profile%2Fsetting-up-and-managing-your-github-profile%2Fcustomizing-your-profile%2Fpersonalizing-your-profile&data=05%7C01%7Crbellet%40ecpi.edu%7C168af546a7cf42795b9408db5df07b5a%7C50656f6f463c40549fc26f4071c6e7fc%7C0%7C0%7C638207059688917082%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=bB9V1jh189ePe1wJzVdgJ5FMtbe8PueuetN7z7BzOS4%3D&reserved=0) with your full name as it appears in your academic affiliation document plus a short bio.

Once you have resolved the issues with your sumbission, [visit us again and reapply](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Feducation.github.com%2F&data=05%7C01%7Crbellet%40ecpi.edu%7C168af546a7cf42795b9408db5df07b5a%7C50656f6f463c40549fc26f4071c6e7fc%7C0%7C0%7C638207059688917082%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=CmL1GeXkYRdqnnR3evQhS76EHLVCgMFYZHGT0R0calY%3D&reserved=0). 
Have questions? Join the [GitHub Education Community](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Forgs%2Fcommunity%2Fdiscussions%2Fcategories%2Fgithub-education&data=05%7C01%7Crbellet%40ecpi.edu%7C168af546a7cf42795b9408db5df07b5a%7C50656f6f463c40549fc26f4071c6e7fc%7C0%7C0%7C638207059688917082%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=RFLnw%2FcfYnbBiAtlcQuFWuFTpkF%2Ba1A9xBF2qrYP1FI%3D&reserved=0) for answers. 
💖 The GitHub Education Team

**Comment by J0J0HA** [comment]
Maybe, if you have a middle name, add it also if it is on you student id. Otherwise, it is some problem at GitHub's side, that you can do nothing about, since manual reviews were discontinued.

**Comment by biozervas** [comment]
I have a similar problem. I have tried updating my github profile, uploading my contract, uploading a picture of my ID and also using my camera to take a picture of my ID on the website. Nothing has worked. I understand that a crucial issue is that my official University ID does not have any dates on it; just my name, affiliation and ID number. Manual reviews have been dropped so I can't even contact someone to ask how I should proceed.

**Reply by MrRichardsDigi** [reply]
Same problem here, incredibly frustrating process.

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
